drfdjango/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
there are some configs that are specific to generic django projects, make sure to remove them, and keep this as a simply rest framework